jobserver/slacks.py

```diff
@@ -99,6 +99,11 @@ def notify_copilots_of_repo_sign_off(repo, channel="co-pilot-support"):
         repo.researcher_signed_off_by.name,
     )
 
-    message = f"The {repo_link} repo was signed off by {user_link}"
+    docs_link = slack.link(
+        "https://bennettinstitute-team-manual.pages.dev/products/opensafely-jobs/#internal-repo-sign-off",
+        "here",
+    )
+
+    message = f"The {repo_link} repo was signed off by {user_link}, docs for our handling this can be found {docs_link}"
```

Comment:
How about something like this, to give a clearer indication about what do to when receiving the slack message?

> The {repo_link} repo was signed off by {user_link}.

> It now needs to be signed off internally (see docs {docs_link}).